operations/fetch.go

```diff
@@ -37,7 +34,6 @@ func Fetch() cli.Command {
 		artifactsFlagName = "artifacts"
 		shallowFlagName   = "shallow"
 		noPatchFlagName   = "patch"
-		tokenFlagName     = "token"
```

Comment:
I think we still need to support cloning over HTTPS with a token (but by shelling out). Are you going to add this later?

Reply:
Does reverting this cause a regression? (Are we using HTTPS with a token somewhere?) If not, I think this should be a separate ticket. I'm trying to keep this scope narrowly to getting people back  fast fetch behavior.

Reply:
It will definitely break docker tests. I doubt anyone else is using this though